apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/api/common/cql/builder/QueryBuilderValuesTest.java

```diff
@@ -131,40 +133,41 @@ public void shouldBindValueModifiers() {
 
   @Test
   public void shouldBindDirectWhereValues() {
-    Query query =
-        new QueryBuilder()
-            .select()
-            .from("ks", "tbl")
-            .where("c1", Predicate.EQ, INT_VALUE1)
-            .where("c2", Predicate.EQ, TEXT_VALUE)
-            .build();
+    Expression<BuiltCondition> expression =
```

Comment:
Would old code not work here? If it would, maybe instead add a new method with new code, but leave old test as-is? This way we would verify backwards-compatibility for other APIs?

If this is not possible, does this mean that existing APIs query building code needs to be changed (I assume not, given that CI passes)

Reply:
yes, CI passes, no impact to old apis.